app/src/main/java/io/zenandroid/onlinego/utils/WhatsNewUtils.kt

```diff
@@ -1,14 +1,35 @@
 package io.zenandroid.onlinego.utils
+import java.security.MessageDigest
 
 object WhatsNewUtils {
 
     val shouldDisplayDialog: Boolean
-            get() = PersistenceManager.lastWhatsNewText != currentText
-//            get() = PersistenceManager.lastWhatsNewText != null && PersistenceManager.lastWhatsNewText != currentText
+        get() = PersistenceManager.lastWhatsNewText != hashString(currentText)
```

Comment:
The reason I did not remove that line is that in the future I do not want everybody to see the whats new when they install the app, but I did want to see it last time (because of the big update). Hence, moving forward, I would like to have it `PersistenceManager.lastWhatsNewText != null &&  PersistenceManager.lastWhatsNewText != hashString(currentText)`. Can we also please rename the field now so that it's obvious it's a hash?

Reply:
ok